staging/src/kubevirt.io/client-go/api/v1/types.go

```diff
@@ -211,6 +211,11 @@ type VirtualMachineInstanceStatus struct {
 	// +optional
 	// +listType=atomic
 	VolumeStatus []VolumeStatus `json:"volumeStatus,omitempty"`
+
+	// FSFreezeStatus is the state of the fs of the guest
+	// it can be either frozen or thawed
+	// +optional
+	FSFreezeStatus string `json:"fsFreezeStatus,omitempty"`
```

Comment:
If this is optional, should probably be `*string`

Reply:
Why is it needed here when it will be in `GuestOSInfo`?

Reply:
@mhenriks from what I see all the other optional variables in this struct which are string are not pointers..
@xpivarc  it is not in the GuestOSInfo, its in the guest agent info which is not shown in the vmi information

Reply:
@ShellyKa13 okay consistency is good